validation-test/stdlib/StringSlicesConcurrentAppend.swift

```diff
@@ -52,12 +59,15 @@ func sliceConcurrentAppendThread(tid: ThreadID) {
     barrier()
 
     // Get a private string.
+    // we *should* be able to change this as we wish without modifying the underlying string.
     var privateString = sharedString
 
     barrier()
 
     // Append to the private string.
-    if tid == .Leader {
+    // This should invoke a deep copy somewhere along the line,
+    // but NOT change the underlying sharedString buffer.
```

Comment:
Sorry, but no, the buffer of `sharedString` will be changed by one of the threads.  Another thread will allocate its own thread.  The reason why the user-visible value of `sharedString` won't change is that its (not shared) length will not change.